project/src/components/containers/offers-list/offers-list.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import Offer from '../../offer/offer';
-import {DefaultPropsType} from '../../../types/propsTypes';
-import {useState} from 'react';
-
-function OffersList({offers}: DefaultPropsType): JSX.Element {
-  const [activeId, setActiveId] = useState('');
+import {OfferType} from '../../../types/mocksTypes';
 
+type OffersListType = {
+  offers: OfferType[];
+  setActiveOffer?: (id: string | undefined) => void;
```

Comment:
```suggestion
  setActiveOffer?: (id?: string )> void;
```
если ожидается в параметре undefined, то параметр можно сделать не обязательным